constants.xml

```diff
@@ -307,6 +307,12 @@
         <DISABLE_SCILLA_LIB>false</DISABLE_SCILLA_LIB>
         <SCILLA_SERVER_PENDING_IN_MS>1500</SCILLA_SERVER_PENDING_IN_MS>
         <SCILLA_SERVER_LOOP_WAIT_MICROSECONDS>10</SCILLA_SERVER_LOOP_WAIT_MICROSECONDS>
+        ## Steps to Enable EVM for a run this temporarily replaces the Scilla Interpreter
```

Comment:
Need clarification this should be committed.

Reply:
Not intentional